src/app/give-help/transport-form/transport-form.component.html

```diff
@@ -0,0 +1,71 @@
+<mat-card>
+  <app-offer-form (submitOffer)="handleSubmit($event)">
+    <div class="row">
+      <div class="col-md-3">
+        <label
+          ><b>{{ "LOCATION_FROM" | translate }}</b></label
+        >
+        <mat-form-field appearance="outline">
+          <input
+            matInput
+            type="text"
+            name="origin"
+            [(ngModel)]="data.origin"
```

Comment:
Please use Reactive forms

Reply:
with form builder: https://angular.io/guide/reactive-forms#using-the-formbuilder-service-to-generate-controls

Reply:
Z jakiego względu? Optował bym za template driven, do przegadaniu na kanale